src/components/utils/weatherVariables.jsx

```
@@ -679,221 +785,257 @@
+            props.page === 'map'
+              ? {
+                  backgroundColor: popupStyles.buttonBg,
+                  color: popupStyles.buttonText,
+                }
+              : { backgroundColor: '#3b82f6', color: 'white' }
+          }
+          onMouseOver={e => {
+            if (props.page === 'map') {
+              e.currentTarget.style.backgroundColor = popupStyles.buttonHoverBg;
+            }
+          }}
```

ESLint

[error] onMouseOver must be accompanied by onFocus for accessibility.

(jsx-a11y/mouse-events-have-key-events)